assets/src/edit-story/components/canvas/framesLayer.js

```diff
@@ -38,7 +38,8 @@ const FramesPageArea = withOverlay(styled(PageArea).attrs({
   className: 'container',
   pointerEvents: 'initial',
 })`
-  background-color: ${({ theme }) => theme.colors.fg.v1};
+  background-color: ${({ theme, backgroundColor }) =>
```

Comment:
Per [this discussion](https://github.com/google/web-stories-wp/pull/383/files#r384696061), please remove `background-color` entirely from here - this is just an oversight.

Reply:
Already done in ef3a047d.